libs/langgraph/langgraph/pregel/remote.py

```diff
@@ -268,6 +309,18 @@ def _sanitize_obj(obj: Any) -> Any:
     def get_state(
         self, config: RunnableConfig, *, subgraphs: bool = False
     ) -> StateSnapshot:
+        """Method to call POST /threads/{thread_id}/state/checkpoint or
```

Comment:
i would add a better description here and then mention which endpoint it calls below. i.e. as a user my intent is not to call the API endpoint but to get the state for a config